### PR TITLE
Added section: added a sister project Phimp.me

### PR DESCRIPTION
**Pull request by saarthakchats**

### CODE REVIEW

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
Provide a test-link :+1:

**Comment by saarthakchats** [comment]
https://saarthakchats.github.io/pattupuparipops/
@0x48piraj

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
image
It's not looking good from my eyes :-1:

**Comment by arjunpandey2003** [comment]
Adjust the margin top and right

**Comment by arjunpandey2003** [comment]
Rest looks good

**Review by NaviRocker — CHANGES_REQUESTED** [review]
I don't think that this section is important here. Let's wait for other reviews.

File: index.html [hunk 1]
```diff
@@ -593,6 +593,32 @@ <h5 class="uppercase italic fade-1-4">A FOSSASIA<br> sister project.”</h5>
...
+			<div class="col-md-4 col-md-offset-1 text-right text-left-xs col-sm-5">
+				<h1 class="uppercase mb24 bold italic">Phimp.me Android<br></h1>
+				<h5 class="uppercase italic fade-1-4">A FOSSASIA<br> sister project.</h5>
+				<hr class="visible-xs">
+			</div>
+			<div class="col-md-5 col-sm-7">
+				<p>
+					Phimp.me is a Photo App for Android that aims to replace proprietary photo applications.
+				</p>
+
```
Comment (supunTE) [inline index.html:613]: Remove unwanted spaces

File: index.html [hunk 2]
```diff
@@ -593,6 +593,32 @@ <h5 class="uppercase italic fade-1-4">A FOSSASIA<br> sister project.”</h5>
...
+				<p>
+					It offers features such as taking photos, adding filters, editing images and uploading them to social networks. Visit <a href="https:/phimp.me/">Phimp Website.</a>
+				</p>
+
```
Comment (supunTE) [inline index.html:617]: This

File: index.html [hunk 3]
```diff
@@ -593,6 +593,32 @@ <h5 class="uppercase italic fade-1-4">A FOSSASIA<br> sister project.”</h5>
...
+				<p>
+					Click beautiful images using the Phimp.me app. Use various advanced scene modes, and variety of balance modes.
+
```
Comment (supunTE) [inline index.html:620]: <p> tag must be closed

**Review by supunTE — CHANGES_REQUESTED** [review]

**Comment by nzec** [comment]
@saarthakchats Not required. It doesn't looks good either.

**Pull request closed without merging**